bundles/org.openhab.binding.yamahamusiccast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="yamahamusiccast"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<thing-type id="device">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>Yamaha MusicCast Model</label>
+		<description>Your Yamaha model with MusicCast functionality</description>
+
+		<channel-groups>
+			<channel-group id="main" typeId="mainControls"/>
+			<channel-group id="zone2" typeId="zone2Controls"/>
+			<channel-group id="zone3" typeId="zone3Controls"/>
+			<channel-group id="zone4" typeId="zone4Controls"/>
+			<channel-group id="playerControls" typeId="playerControls"/>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Address</label>
+				<context>network-address</context>
+				<description>The IP address of the AVR to control.</description>
+			</parameter>
+			<parameter name="syncVolume" type="boolean">
+				<label>Sync Volume</label>
+				<description>Sync Volume across linked Music Cast models</description>
+				<default>true</default>
+			</parameter>
+			<parameter name="defaultAfterMCLink" type="text">
+				<label>MC Link Default Value</label>
+				<description>Default value for client when MC Link is broken</description>
+				<default>net_radio</default>
+				<options>
+					<option value="none">None</option>
+					<option value="net_radio">Net Radio</option>
+				</options>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-group-type id="mainControls">
+		<label>Main Zone (default)</label>
+	</channel-group-type>
+
+	<channel-group-type id="zone2Controls">
+		<label>Zone 2</label>
+	</channel-group-type>
+
+	<channel-group-type id="zone3Controls">
+		<label>Zone 3</label>
+	</channel-group-type>
+
+	<channel-group-type id="zone4Controls">
+		<label>Zone 4</label>
+	</channel-group-type>
+
+	<channel-group-type id="playerControls">
+		<label>player Controls</label>
+		<channels>
+			<channel id="player" typeId="player"/>
+			<channel id="artist" typeId="artist"/>
+			<channel id="track" typeId="track"/>
+			<channel id="album" typeId="album"/>
+			<channel id="albumArt" typeId="albumArt"/>
+			<channel id="repeat" typeId="repeat"/>
+			<channel id="shuffle" typeId="shuffle"/>
+			<channel id="playTime" typeId="playTime"/>
+			<channel id="totalTime" typeId="totalTime"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="volumeAbs">
+		<item-type>Number</item-type>
+		<label>Volume</label>
+		<description>Volume channel - Absolute value</description>
+	</channel-type>
```

Comment:
You could use the predefined channel type. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types

Reply:
This is value from the device itself which can range from -80 to 80
The channel Volume is already set so system.volume. The value for the percentage is recalculated by the binding.
No need to change this one as well.